KOGITO-4724 sql script bundle note (#3835)
kiegroup/kie-docs: doc-content/kogito-docs/src/main/asciidoc/release-notes/chap-kogito-release-notes.adoc

quarkus.datasource.password=kogito-pass
------

==== Persistence script bundle

> Persistence setup scripts both for {PRODUCT} runtimes and supporting services like data index, etc. are bundled as the kogito-ddl-{maven_artifact_version}-db-scripts.zip artifact.

Reviewer: ```suggestion
Persistence setup scripts for {PRODUCT} runtimes and {PRODUCT} runtimes supporting services are now included in the kogito-ddl-{maven_artifact_version}-db-scripts.zip artifact.
```
